loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -1,21 +1,47 @@
-[*
-**  PowerOn Name:       BANNO.LOANPAYMENT.SKIP.V1.POW
-**  Letterfile Name:    BANNO.LOANPAYMENT.SKIP.CFG
+[*  Copyright 2019-2022 Jack Henry and Associates
 **
-**  Copyright 2019 Jack Henry and Associates
-**
-**  03/24/2020: fixes for no fee and skip period
-**  04/16/2020: fixes for invalid punctuation in share descriptions
-**  06/16/2020: Correct new due date and maturity date to insure
-**              it falls on the correct day of the week.
-**  10/28/2021: Correct processing of disclosure blank lines. Added
-**              forced space between consecutive lines of text
-**  11/30/2021: TKainz - Banno
-**              Changed max DQ grace days from 30 to unlimited
+**  PowerOn Name:                   BANNO.LOANPAYMENT.SKIP.V1.POW
+**  Configuration Letterfile Name:  BANNO.LOANPAYMENT.SKIP.CFG
 **
+**  Program Summary
+**  --------------------------------------------------------------------
 **  This Banno service PowerOn allow the user to select one or more
 **  eligible loans and, for a fee, have the loan due date advanced
 **  by a month.
+
+**
+**  Modification History
+**  --------------------------------------------------------------------
+**  Ver. 1.0.0  Original Programming
+**  Ver. 1.0.1  03/24/2020: fixes for no fee and skip period
+**  Ver. 1.0.2  04/16/2020: fixes for invalid punctuation in share
+**              descriptions
+**  Ver. 1.0.3  06/16/2020: Correct new due date and maturity date to
+**              insure it falls on the correct day of the week.
+**  Ver. 1.0.4  10/28/2021: Correct processing of disclosure blank lines.
+**              Added forced space between consecutive lines of text
+**  Ver. 1.0.5  11/30/2021: TKainz - Banno
+**              Changed max DQ grace days from 30 to unlimited
+**  Ver 1.1.0 0 1/24/2021: TKainz - Banno

```

Comment:
2022

Reply:
Arrgghhh.... My first 2021 of the 2022 year!!  Fixed